setup.sh

```diff
@@ -348,8 +348,14 @@ download_db_dump         # pre-req: install_deps
 create_pg_databases      # pre-req: install_deps
 create_default_keeper_config # pre-req: update_userinfo
 
-# We want to run this only with the brew version of python, NOT OSX's python3
-install_keeper $(brew --prefix)/bin/python3
+# If brew is installed, run this only with the brew version of python, NOT OSX's python3
+if [ -n "$(which brew)" ]; then
+    echo "installing with brew"
```

Comment:
(This isn't really accurate. It's not using brew to install, it's using brew's python3.  Personally, I'd just take out the echo statements.)

Reply:
Great point. These may or may not have been for debugging and I hastily forgot to remove them... 😑